docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -166,248 +166,170 @@ You need to perform the following steps:
+  docker pull tianon/postgres-upgrade:11-to-14
+  docker pull nginxinc/nginx-unprivileged:1.23.3
+  ```
 
 1. Login to your target container registry, as per the following example that uses Google Container Registry (GCR):
 
-   ```sh
-   docker login -u _json_key --password-stdin https://gcr.io < .ssh/my-service-account-key.json
-   ```
+  ```sh
+  docker login -u _json_key --password-stdin https://gcr.io < .ssh/my-service-account-key.json
+  ```
 
 1. Tag the local images to your target registry, as follows:
 
-   ```sh
-   docker images 
-   ```
-
-   ```output
-   REPOSITORY              TAG                           IMAGE ID    CREATED     SIZE
-   quay.io/yugabyte/yugaware      2.5.1.0-b153               **a04fef023c7c**  6 weeks ago   2.54GB
-   quay.io/yugabyte/thirdparty-deps   latest                   **721453480a0f**  2 months ago  447MB
-   nginx                1.17.4                          **5a9061639d0a**  15 months ago  126MB
-   postgres               11.5                          **5f1485c70c9a**  15 months ago  293MB
-   prom/prometheus           v2.2.1                         **cc866859f8df**  2 years ago   113MB
-   janeczku/go-dnsmasq         release-1.0.7                      **caef6233eac4**  4 years ago   7.38MB
-   ```
-
-   ```sh
-   docker tag a04fef023c7c gcr.io/dataengineeringdemos/yugabyte/yugaware:2.5.1.0-b153
-   docker tag 721453480a0f gcr.io/dataengineeringdemos/yugabyte/thirdparty-deps:latest
-   docker tag 5a9061639d0a gcr.io/dataengineeringdemos/yugabyte/nginx:1.17.4
-   docker tag 5f1485c70c9a gcr.io/dataengineeringdemos/yugabyte/postgres:11.5
-   docker tag cc866859f8df gcr.io/dataengineeringdemos/prom/prometheus:v2.2.1
-   docker tag caef6233eac4 gcr.io/dataengineeringdemos/janeczku/go-dnsmasq:release-1.0.7
-   ```
+  ```sh
+  docker images
+  ```
+
+  ```output
+  REPOSITORY                         TAG            IMAGE ID       CREATED        SIZE
+  tianon/postgres-upgrade            11-to-14       e4e13bbe3446   2 days ago     432MB
+  quay.io/yugabyte/yugaware          2.16.1.0-b50   230d619f5c20   2 days ago     2.56GB
+  nginxinc/nginx-unprivileged        1.23.3         63e2b5caa055   3 weeks ago    142MB
+  quay.io/yugabyte/thirdparty-deps   latest         479490b66ad9   4 weeks ago    665MB
+  prom/prometheus                    v2.41.0        932c2dbe7d3e   6 weeks ago    231MB
+  postgres                           14.4           e09e90144645   6 months ago   376MB
+  ```
+
+  ```sh
+  docker tag 230d619f5c20 gcr.io/dataengineeringdemos/yugabyte/yugaware:2.16.1.0-b50
+  docker tag 479490b66ad9 gcr.io/dataengineeringdemos/yugabyte/thirdparty-deps:latest
+  docker tag 63e2b5caa055 gcr.io/dataengineeringdemos/yugabyte/nginxinc/nginx-unprivileged:1.23.3
+  docker tag e09e90144645 gcr.io/dataengineeringdemos/yugabyte/postgres:14.4
+  docker tag e4e13bbe3446 gcr.io/dataengineeringdemos/tianon/postgres-upgrade:11-to-14
+  docker tag 932c2dbe7d3e gcr.io/dataengineeringdemos/prom/prometheus:v2.41.0
```

Comment:
I think we don't need to do `docker images` and then use the image id to tag it. 
Something like `docker tag postgres:14.4 gcr.io/demo/postgres:14.4` should just work.
Maybe try it once before making the change?